tests/unit/BlockStorage.d

```diff
@@ -72,7 +72,7 @@ private void main ()
             ]
         );
         tx.inputs[0].signature = gen_key_pair.secret.sign(hashFull(tx)[]);
-        blocks ~= makeNewBlock(blocks[$ - 1], [tx]);
+        blocks ~= makeNewBlock(blocks[$ - 1], [tx], null);
```

Comment:
```suggestion
        blocks ~= makeNewBlock(blocks[$ - 1], [tx]);
```

Reply:
applied the suggestion.

Reply:
Where do `makeNewBlock (blocks [$ - 1], [tx])` look?

Reply:
Oh sorry I didn't realize it was in `tests/unit`. So yeah, revert to `null`.

Reply:
Oh okay. I was confused for a while.